src/spikeinterface/preprocessing/average_across_direction.py

```diff
@@ -24,12 +24,12 @@ def __init__(
         ----------
         parent_recording : BaseRecording
             recording to zero-pad
-        direction : str
+        direction : str, default 'y'
```

Comment:
What are the options? (ie can someone put in 'x' or 'z')?

Reply:
yes, will add the options